### DISCUSSION TITLE
no vm/vps supported

### DISCUSSION THREAD

**Discussion by Tjenn920** [community/community#34531] [Other Feature Feedback, Questions, & Ideas]
Hello everyone im new, nice to meet you guys. I am trying to load a file using windows 10 from git but I am getting the error no vm/vps supported. Can someone kindly tell me what that means and what I need to do?

**Comment by jfullstackdev** [comment]
what specific file is that ? that seems related to a virtual machine

if you can have screenshots, that's better
